app/controllers/phylo_trees_controller.rb

```diff
@@ -37,6 +37,7 @@ def index
     @taxon = {}
 
     taxid = params[:taxId]
+    return if HUMAN_TAX_IDS.include? taxid.to_i
```

Comment:
I'm a little concerned how the front-end will respond to these empty responses. It would be great if it displayed error messages or something. I just checked and `PhyloTreeListView` can crash if no trees are provided, but that's a separate bug I can address.

For this PR, can you return something like `render json: { status: :not_acceptable, message: "Human taxids are not allowed"}` instead of returning nothing.

Reply:
ok sounds good